client/packages/common/src/utils/quantities/quantities.ts

```diff
@@ -15,4 +15,20 @@ export const QuantityUtils = {
 
     return Math.max(suggested, 0);
   },
+
+  /** Converts a number of packs to dose quantity */
+  packsToDoses: (
+    numPacks: number,
+    line: { packSize: number; dosesPerUnit?: number }
+  ) => {
+    return NumUtils.round(numPacks * line.packSize * (line.dosesPerUnit || 1));
```

Comment:
Should we not be rounding to 2 dp and showing tooltip like the rest of the app...?

Reply:
Doses are always a whole number, only time you get decimals is bc we store decimal number of packs - like if pack size was 3, then each dose would be 0.333... end up with 0.00000001 = 0.00001 doses at the end, but actually have 0 👀

Reply:
Ah, thought children got same vaccine but in smaller doses, but if it's whole then that's fine~

Reply:
If they do, we don't support 👀 All our dose inputs are integer only...